app/components/dashboard/all_collections_row_component.rb

```diff
@@ -0,0 +1,32 @@
+# typed: true
+# frozen_string_literal: true
+
+module Dashboard
+  # Renders a list of all collections
+  class AllCollectionsRowComponent < ApplicationComponent
+    sig { params(collection: Collection, counts: Hash).void }
+    def initialize(collection:, counts:)
+      @collection = collection
+      @counts = counts
+    end
+
+    attr_reader :collection, :counts
+
+    def draft?
+      return true if @collection.head&.state&.include? 'draft'
+
+      false
+    end
+
+    def draft_label
+      case @collection.head&.state
+      when 'first_draft'
+        return '- Draft'
```

Comment:
These returns aren't necessary if this is the last statement in the method. (e.g. get rid of the `nil`)